### DISCUSSION TITLE
Decrypted connection?

### DISCUSSION THREAD

**Discussion by marcussacana** [vpnhood/VpnHood#61] [Q&A] [answered]
My purpose using a VPN isn't for any type of privacy, I need just access my LAN devices over the internet, since only my computer have an public IP in my home network.

With that said, I want the best performance possible to the connection, regardless of the connection transparency, is possible do a decrypted connection? Or maybe someone can hint me where in the source is the encrypt/decrypt functions to me just cut out the encryption?

**Comment by trudyhood** [comment, marked as answer]
Yes, VpnHood is not suitable for your needs; it deliberately doesn't route local networks and delegates DNS packets to public Google DNS servers. I can add an option to VpnHood to route Local Networks, but I am curious what makes you end up VpnHood. Let's not reinvent the wheel. VpnHood is optimized to bypass and circumvent a kind of firewalls; you need a regular VPN designed to connect you to your Local Network via the internets, such as OpenVPN, SoftEther, or Windows build in VPN.

By the way, thank you for trying VpnHood; I am sorry that it doesn't do what you are looking for.

**Comment by marcussacana** [comment]
I will basically use for RemotePlay in my PS4 out of home, this require a very low ping connection and with a good speed as well.
My home internet only allow we have public ipv6 address, our ipv4 address is shared with all users of my internet provider then I can't redirect the ports to my devices. And to solve that I started look for open a VPN server locally and connect with ipv6 to him, since PS4 don't support ipv6 at all. 
Initially I wanted try use the embbed windows PPTP server, but not sure why, but my windows just don't seems to open the VPN server.
Anyway, The OpenVPN worked, but had poor perfomance to me.
After that I found this VPNHood, that in my tests got a better perfomance than the PPTP and OpenVPN, then I really wanted use it.
But seems that don't will be possible.
While allow local networks option will be cool, I doesn't want to bother you.

**Reply by trudyhood** [reply]
This option already exists, and I need to add it to UI. You've intrigued in your case. But, I am not sure how Remote Play can find your device. There is also an issue for running the VpnHood server on IPv6 that will be solved soon. Let's try it after we solve this issue and add that feature. Meanwhile, you are welcome to share any idea that pops up in your mind.

I added an issue for it, 
https://github.com/vpnhood/VpnHood/issues/62

**Reply by marcussacana** [reply]
Thank you :) The remote play have an unofficial open source client called "chiaki" that allow do the remote play connection manually with ipv4, I just really need make accessible my console over the network to be able to connect.

**Reply by trudyhood** [reply]
Hi, 
We have released a 2.2.283 with the options you needed. I am with you to go deeper into  the issue you have. I really like make it works.

**Reply by marcussacana** [reply]
thank you so much :)
